cli.js

```diff
@@ -77,6 +77,8 @@ function runFastify (opts) {
 
   if (opts.address) {
     fastify.listen(opts.port, opts.address, listen)
+  } if (opts.socket) {
```

Comment:
`else if`?

Reply:
Wow! strange... `standard` says nothing about it!